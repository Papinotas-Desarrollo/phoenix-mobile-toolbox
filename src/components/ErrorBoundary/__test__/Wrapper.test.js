import { shallow } from 'enzyme';
import * as React from 'react';
import { ErrorBoundaryWrapper } from '../';

describe('Snapshot test ::', () => {
  it('With props snapshot', () => {
    const props = {
      secure: '',
      styles: {},
      children: [],
    };
    const tree = shallow(
      <ErrorBoundaryWrapper {...props}>
        <React.Fragment />
      </ErrorBoundaryWrapper>
    );
    expect(tree).toMatchSnapshot();
  });
});
