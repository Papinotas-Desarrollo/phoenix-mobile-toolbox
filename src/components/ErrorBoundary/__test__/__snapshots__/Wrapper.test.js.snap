// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test :: With props snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Wrapper
    secure=""
    styles={Object {}}
  >
    <React.Fragment />
  </Wrapper>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "FallbackComponent": [Function],
      "children": <React.Fragment />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": Symbol(react.fragment),
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "FallbackComponent": [Function],
        "children": <React.Fragment />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": Symbol(react.fragment),
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
